src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -292,17 +292,19 @@ void AcousticWaveEquationSEM::addSourceToRightHandSide( integer const & cycleNum
   } );
 }
 
-
-void AcousticWaveEquationSEM::computeSeismoTrace( real64 const time_n, real64 const dt, localIndex const iSeismo, arrayView1d< real64 > const pressure_np1, arrayView1d< real64 > const pressure_n )
+void AcousticWaveEquationSEM::computeSeismoTrace( real64 const time_n,
+                                                  real64 const dt,
+                                                  real64 const timeSeismo,
+                                                  localIndex iSeismo,
+                                                  arrayView1d< real64 > const var_at_np1,
+                                                  arrayView1d< real64 > const var_at_n,
+                                                  arrayView2d< real64 > var_at_receivers )
```

Comment:
```suggestion
                                                  arrayView1d< real64 const > const var_np1,
                                                  arrayView1d< real64 const > const var_n,
                                                  arrayView2d< real64 > const varAtReceivers )
```
just to be consistent with the style convention used in the other parts of the code, where we use `time_n`, `density_n`, etc for the variables evaluated at time `n`. Also, whenever the variable name does not explicitly mention the time level (`_n`, etc), we try to use camelCase, hence `varAtReceivers`.

Also, I suggest to change the template from `real64` to `real64 const` for the two `arrayView1d` to clearly state that this function does not modify their values.

Reply:
style changed with the last commits, I also added the const keywords as required (you are right, it is good practice)